src/components/headersEditor/HeadersEditor.tsx

```diff
@@ -0,0 +1,109 @@
+import { CloseSquareFilled, DeleteOutlined, PlusOutlined } from '@ant-design/icons';
+import { AutoComplete, Button, Checkbox, Divider, Form, Space } from 'antd';
+import React, { useState } from 'react';
+
+import { defaultHeaderValues } from '@/constants/defaultHeaderValues';
+import { Header, HeadersEditorProps, HeadersFormValues } from '@/types/headersSectionTypes';
```

Comment:
```suggestion
import type { Header, HeadersEditorProps, HeadersFormValues } from '@/types/headersSectionTypes';
```
Серёжа советовал для импорта типов использовать `import type` 🤓 🤓🤓🤓🤓🤓🤓

Reply:
а я уже добавила правило в еслинт и он сам все поправит после мержа моей ветки

Reply:
ну почти сам
он покричит и мы поправим)